slave.mk

```diff
@@ -973,7 +973,6 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
                 $(LIBNSS_TACPLUS) \
                 $(MONIT) \
                 $(OPENSSH_SERVER) \
-		$(SMARTMONTOOLS) \
```

Comment:
Think you may have a tab / space issue here in the diff.

Reply:
I checked the original file, seems that there is a leading Tab and a space in the line of "$(SMARTMONTOOLS)", for other lines they are no Tab but all spaces. Anyway, I think this is not relevant anymore since this line will be deleted.

Reply:
Whoops sorry about that, commented on instinct there.